docker/entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -eu
 
+# If typos does not exist, download and extract it
+if [[ ! -f /usr/local/bin/typos ]]; then
```

Comment:
Why is this needed?  It caches the location we download to?

If so, when does it invalidate? If we have a new release will users be sure to get the new version?

Reply:
It doesn't caches anything. It was initially added because if you don't download typos within docker, you need to download it within the shell.

However, later it was discovered that there was already an if statement checking if the `typos` command exists, so they were merged together.